test/integration_test.py

```diff
@@ -351,9 +351,10 @@ def _wait_for_indexer():
                                           bundle_fqids=catalog.bundle_fqids)
 
         for catalog in catalogs:
-            self._test_manifest(catalog.name)
-            self._test_dos_and_drs(catalog.name)
-            self._test_repository_files(catalog.name)
+            if config.catalogs[catalog.name].plugins['repository'].name != 'local':
```

Comment:
IIRC, the `local` plugin is in the attic so we shouldn't have this conditional in active code.

Reply:
It was re-added for this PR in a `drop!` so I could repeat the tests that used it. I've now dropped the `drop!` commits.